cmd/charm-ingest/whitelist_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the GPLv3, see LICENCE file for details.
+
+package main
+
+import (
+	"strings"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/juju/charmstore-client/internal/ingest"
+	"gopkg.in/juju/charmrepo.v4/csclient/params"
+)
+
+var whiteListTests = []struct {
+	tag         string
+	whitelist   string
+	expect      []ingest.WhitelistEntity
+	expectError string
+}{{
+	tag: "valid_whitelist",
```

Comment:
I've taken to calling this `testName` recently. Otherwise our other convention is `about`, although that makes a bit less sense since it doesn't have whitespace in anymore.